snowexsql/tables/measurement_type.py

```diff
@@ -21,7 +23,7 @@ class HasMeasurement:
 
     @declared_attr
     def measurement_id(cls):
-        return Column(Integer, ForeignKey('public.measurements.id'),
+        return Column(Integer, ForeignKey('public.measurement_type.id'),
                       index=True)
 
     @declared_attr
```

Comment:
Also change this definition to 
```python
def measurement_type
```

Reply:
assuming here you mean
```python
def measurement_type_id
```
for consistency with other foreign key names.